ggml/src/ggml.c

```diff
@@ -5369,6 +5370,7 @@ static struct ggml_tensor * ggml_group_norm_impl(
     struct ggml_tensor * result = inplace ? ggml_view_tensor(ctx, a) : ggml_dup_tensor(ctx, a);
 
     result->op_params[0] = n_groups;
+    *(float*)(result->op_params + 1) = eps;
```

Comment:
```suggestion
    ggml_set_op_params_i32(result, 0, n_groups);
    ggml_set_op_params_f32(result, 1, eps);
```

I think this should do the same thing